src/python/bot/tasks/commands.py

```diff
@@ -49,7 +49,7 @@
     'fuzz': fuzz_task,
     'impact': impact_task,
     'minimize': minimize_task,
-    'ml_train': ml_train_task,
+    'rnn_generator': rnn_generator_task,
```

Comment:
Can you also add an entry for train_gradientfuzz? We shouldn't add the full file for it but I'd still like to have a simple dummy that logs that it's running so that we can confirm things work before landing the future changes.

Reply:
Yeah -- sounds good. I'll add the dummy as well!